mods/bones/init.lua

```diff
@@ -214,21 +220,19 @@ minetest.register_on_dieplayer(function(player)
 	local meta = minetest.get_meta(pos)
 	local inv = meta:get_inventory()
 	inv:set_size("main", 8 * 4)
-	inv:set_list("main", player_inv:get_list("main"))
 
-	for i = 1, player_inv:get_size("craft") do
-		local stack = player_inv:get_stack("craft", i)
-		if inv:room_for_item("main", stack) then
-			inv:add_item("main", stack)
-		else
-			--drop if no space left
-			drop(pos, stack)
+	for _, list_name in ipairs(player_inventory_lists) do
+		for i = 1, player_inv:get_size(list_name) do
+			local stack = player_inv:get_stack(list_name, i)
```

Comment:
You mean something like `inv:add_item("main", ItemStack(""))`?
By design i'd imagine [nothing](https://github.com/minetest/minetest/blob/ae7fb9b5bea291f81a82912d42fc43b7e71702bf/src/inventory.cpp#L328).

I did see some dropped "empty" ItemStacks with the chainsaw the other day, so maybe there is some corner case, that however needs to be fixed engine wide then. What i'm assuming are empty ItemStacks, did not cause any further trouble, just some confusion.
![handdrop](https://cloud.githubusercontent.com/assets/1937987/18521041/cdf781ba-7aaa-11e6-86d7-15e4f878bbe1.png)